include/solvers/mf_navier_stokes_operators.h

```diff
@@ -410,6 +423,18 @@ class NavierStokesOperatorBase : public Subscriptor
    */
   std::shared_ptr<SimulationControl> simulation_control;
 
+  /**
+   * @brief Flag to turn hessian terms from jacobian on or off.
+   *
+   */
+  bool enable_hessians_jacobian;
+
+  /**
+   * @brief Flag to turn hessian terms from residual on or off.
+   *
+   */
+  bool enable_hessians_residual;
```

Comment:
You could use /// instead here. I don't think  you need a brief because this is not a function

Reply:
That's true... but all this class was documented before we knew that we could use ///. So, if you are okay, I will leave it like this and then update all the files that have member variables documented in this fashion in one PR.

Reply:
Sure thing.  I'm ok with this